ci/run_integration_test.sh

```diff
@@ -1,6 +1,6 @@
 #! /bin/bash -e
 
-BRANCH='branch-4.2'
+BRANCH='master'
```

Comment:
It is bad idea to run it on master since we won't know where the problem is, on scylla side or on driver side.
Let's target 4.5 or 4.6.

Reply:
thanks!
you're, I'll change it to 4.6